src/renderer/components/dock/pod-logs.tsx

```diff
@@ -185,6 +188,7 @@ export class PodLogs extends React.Component<Props> {
     const { searchQuery, isActiveOverlay } = searchStore;
     const item = this.logs[rowIndex];
     const contents: React.ReactElement[] = [];
+    const ansiToHtml = (ansi: string) => DOMPurify.sanitize(this.colorConverter.ansi_to_html(ansi));
```

Comment:
Why is this a local variable and not a new method?

Reply:
This function only used inside wrapper method it's defined. No external places using it.

Reply:
Ah okay